main.go

```diff
@@ -1,11 +1,35 @@
 package main
 
 import (
-	"net/http"
-	"url-shortener/routes"
+	"log"
+	"os"
+
+	"tiny-site-backend/initializers"
+	"tiny-site-backend/routes"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/cors"
+	"github.com/gofiber/fiber/v2/middleware/logger"
 )
 
+func init() {
+	config, err := initializers.LoadConfig(".")
+	if err != nil {
+		log.Fatalln("Failed to load environment variables! \n", err.Error())
+	}
+	initializers.ConnectDB(&config)
+}
+
 func main() {
-	http.HandleFunc("/", routes.MainHandler)
-	http.ListenAndServe(":8080", nil)
+	Origin := os.Getenv("DOMAIN")
+	app := fiber.New()
+	app.Use(logger.New())
+	app.Use(cors.New(cors.Config{
+		AllowOrigins:     Origin,
+		AllowHeaders:     "Origin, Content-Type, Accept",
+		AllowMethods:     "GET, POST",
+		AllowCredentials: true,
+	}))
+	routes.SetupRoutes(app)
+	log.Fatal(app.Listen(":8000"))
```

Comment:
This is the log just for printing the info, fatal might not be the right one to log

Reply:
removed